Rhmap 9094 add travis file

mikenairn opened the pull request:
* Adds a travis build file, runs pep8 and unit tests
* Executed autopep8 on all existing plugins (no functional changes)
* Add build status to readme

Jira:

https://issues.jboss.org/browse/RHMAP-9094

pmccarthy commented on README.md:
```diff
@@ -1,3 +1,5 @@
 # RHMAP Nagios for Docker
 
+[![Build Status](https://travis-ci.org/feedhenry/nagios-container.svg?branch=master)](https://travis-ci.org/feedhenry/nagios-container)
```
Is it also worth adding in a section which outlines the two validation commands being run as part of the travis build? i.e. pep8 plugins/ --ignore E501 --show-source and python -m unittest discover -s plugins/default so that people know how to run this tests locally beforehand.

mikenairn replied:
Yes it would, let me update this.

pmccarthy commented:
Overall this looks fine. Will be nice to have a style guide in place to ensure consistency.

mikenairn commented:
@pmccarthy Added some test and style verification instructions to the readme caddd23b3bbef2f43e989a41674c5d660e35785a

pmccarthy commented on README.md:
````diff
@@ -1,3 +1,39 @@
+To verify:
+
+```
+pep8 plugins/ --show-source
````
@mikenairn  Ran through the commands in this guide, for the pep8 command, a bunch of E501 errors were displayed for me locally. I know these are ignored as part of the travis build itself, but could be confusing to anyone that is unaware of this. Maybe a note about E501 would be helpful

mikenairn replied:
Updated to make the command the same. The  E501 error is about line lengths, you can make it allow longer lines but I think its safe enough to ignore them all for now.

mikenairn commented:
@pmccarthy Change of plan, I added a setup.cfg and modified the allowed line length for pep8 checks. This config gets shared by both pep8 and autopep8 so seems the best way to configure it.

pmccarthy commented:
@mikenairn Went through the readme again, can confirm that all commands are working as expected now, no more E501 errors which verifies the setup.cfg property is working. Also ran this version of the nagios container and can confirm that all checks are passing as expected.
